src/components/App/AppIFrame.js

```diff
@@ -182,6 +182,8 @@ class AppIFrame extends React.Component {
         {progressBar}
         <StyledIFrame
           name="AppIFrame"
+          allow="camera; microphone"
+          allowusermedia
```

Comment:
Reading the [spec on `allowusermedia`](http://wicg.github.io/feature-policy/#iframe-allowusermedia-attribute), it seems like we don't need it since `allow="camera; microphone"` supersedes it.

However, it looks like [`allow`'s not very compatible yet](https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy#Browser_compatibility) so I wonder if `allowusermedia` is more compatible at this point. I haven't been able to find compatibility tables for `allowusermedia`, though.

Reply:
That was my thinking but actually it looks like allowusermedia is pretty new too and Chromium at least implemented support for both attributes at the same time last year so it probably makes sense to just remove allowusermedia:

https://bugs.chromium.org/p/chromium/issues/detail?id=682258

That being said, I think this pull doesn't actually work! I had thought when I was testing it out that adding allow="camera" was letting me access the camera in our Aragon app but I think at some point I had removed the sandbox attribute in my linked aragon wrapper and didn't test properly with it turned back on. 

Possibly relevant Firefox tracker: https://bugzilla.mozilla.org/show_bug.cgi?id=1389198

I'm still not sure why it's not working in Chromium. I'll try to investigate a bit more so maybe hold off on merging this for now.

Reply:
@sohkai I have a feeling that the allow attribute should work in Chromium if https is enabled but I haven't figured out how to run aragon locally over https to test it ou (I tried setting up a proxy with nginx but it didn't like that the websocket and ipfs connections weren't secure). Would you be able to confirm and then possibly merge this for us? I took off the allowusermedia attribute.

Reply:
@asoltys I think you might've been bit by the allowlist's origin.

Reading the [spec](http://wicg.github.io/feature-policy/#iframe-allow-attribute), the default origin for the iframe attribute is "src", which represents the origin of the iframe's src attribute. However, we explicitly don't include `allow-same-origin` in the iframe's sandboxing, rendering the subframe's origin to be completely detached.

Could you try again with `allow="camera *; microphone *"`?

------

AFAIK this shouldn't be related to https, but maybe I missed something in the spec that only allows this in secure origins?